Marlin/Marlin_main.cpp

```diff
@@ -248,8 +248,11 @@ bool Stopped = false;
 bool CooldownNoWait = true;
 bool target_direction;
 
-#ifdef ENABLE_AUTO_BED_LEVELING
+#if defined(ENABLE_AUTO_BED_LEVELING) || defined(EASY_UI)
```

Comment:
The fact is that i just want to use this conditional to use xy_travel_speed on do_blocking_move_to(). So you can use and easy_ui, with or without auto_bed_leveling enable. but thanks for the idea! ;)